### DISCUSSION TITLE
Draggable table row

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#3263] [Q&A] [answered]
Hi team, I'm doing Paste migration on some Segment pages. One of the tables in Segment Identity Resolution has a "draggable table row" feature where a user can change the order of the table row to indicate the identifier priority. Defining the priority of the  identifier is one of the most important steps of setting up the Identify Resolution strategy. For context, user generally have ~10 identifiers in the table

Does Paste have something equivalent and what's your recommendation moving forward? Help is appreciated!

screen recording attached

https://github.com/twilio-labs/paste/assets/56697620/2dba5f6c-14f8-4ace-89a3-0854afb6990c

**Comment by TheSisb** [comment, marked as answer]
Hi @charleneyjh1,

We don't currently have this in our system. Let me know if this previous reply from Simon on another thread helps clarify: https://github.com/twilio-labs/paste/discussions/3221

**Reply by charleneyjh1** [reply]
Thank you so much @TheSisb ! That answers my question
